### PR TITLE
Refactor/docs remove commercial

### PR DESCRIPTION
**Pull request by GuillaumeTech**
remove commercial parts in the docs

If the feature is a refactor, please explain why your way is better

### CODE REVIEW

File: docs/guide/rasa/custom_actions.mdx [hunk 1]
```diff
@@ -48,92 +48,9 @@ In the `run` method you can access the conversation state and utter responses. T
...
-import actions.my_module
-```
-
-</Important>
-
-#### Production
-
-Contact us if you want us to host your production actions deployment.
-
-### Botfront Open Source
```
Comment (GuillaumeTech) [inline docs/guide/rasa/custom_actions.mdx:136]: I removed that quickly as it was mentioning Botfront cloud But I'm wondering If I should bring it back and reword it a bit for os ?

File: docs/guide/rasa/stories/intents_and_entities.mdx [hunk 2]
```diff
@@ -93,9 +93,7 @@ Here is an example of duckling configuration:
 
 ```yaml
 - name: DucklingHTTPExtractor
-  # Specify a URL if using Botfront open source or enterprise edition.
-  # Botfront Cloud users must not specify a URL
-  # url: http://duckling:8000
```
Comment (GuillaumeTech) [inline docs/guide/rasa/stories/intents_and_entities.mdx:98]: I'm not sure what to do about duckling

**Comment by GuillaumeTech** [comment]
Should we mention the terraform  somewhere ? @znat

File: docs/guide/rasa/stories/intents_and_entities.mdx (hunk 2 above)
Comment (znat) [reply docs/guide/rasa/stories/intents_and_entities.mdx:98]: So it's ok

**Review by znat — APPROVED** [review]

**Pull request merged**